packages/web3-eth-iban/package.json

```diff
@@ -11,13 +11,19 @@
     "scripts": {
         "build": "rollup -c",
         "dev": "rollup -c -w",
-        "test": "jest"
+        "test": "npm run-script dtslint && jest",
+        "dtslint": "dtslint types"
     },
+    "types": "types",
     "dependencies": {
         "bn.js": "4.11.8",
         "web3-utils": "1.0.0-beta.36"
     },
     "files": [
-        "dist"
-    ]
+        "dist",
+        "types/index.d.ts"
+    ],
+    "devDependencies": {
+        "dtslint": "^0.3.0"
```

Comment:
This dependency should only be on the root folder. (web3.js/package.json)

Reply:
dtslint will not run if its in the parent package.. it seems dtslint can only run if it's in the node_modules of the current package. 

You get `'dtslint' is not recognized as an internal or external command,` if you try.